qiita_db/test/test_artifact.py

```diff
@@ -496,6 +497,87 @@ def test_parents(self):
         exp_parents = [qdb.artifact.Artifact(2)]
         self.assertEqual(qdb.artifact.Artifact(4).parents, exp_parents)
 
```

Comment:
Would be good to add a test in for `_create_lineage_graph_from_edge_list` explicitly.